src/foremast/utils/templates.py

```diff
@@ -25,7 +26,7 @@
 LOG = logging.getLogger(__name__)
 
 HERE = os.path.dirname(os.path.realpath(__file__))
-LOCAL_TEMPLATES = '{0}/../templates/'.format(HERE)
+LOCAL_TEMPLATES = str(pathlib.Path('{0}/../templates/'.format(HERE)).resolve())
```

Comment:
Instead of stringifying as we go, we should wait until the variable is used to stringify.

Reply:
Refactored.